libsolidity/formal/SMTEncoder.cpp

```diff
@@ -511,23 +338,11 @@ void SMTChecker::endVisit(UnaryOperation const& _op)
 				"Assertion checker does not yet implement such increments / decrements."
 			);
 
-		addVerificationTarget(
```

Comment:
Wasn't this moved already in a previous PR?

Reply:
The previous PR changed `addOverflowTarget` to `addVerificationTarget`, but still in the same file.